src/main/webapp/resources/js/pages/analyses/analysis-outputs-table.js

```diff
@@ -1,369 +1,371 @@
-// import { Grid } from "ag-grid-community/main";
```

Comment:
File comment

Reply:
This is being redone in a different branch, not worth it here

Reply:
Sounds good!